config/config_params.go

```diff
@@ -109,7 +109,8 @@ type Config struct {
 	Ipv6Support    bool `config:"bool;true"`
 	IgnoreLooseRPF bool `config:"bool;false"`
 
-	IptablesRefreshInterval int `config:"int;10"`
+	IptablesRefreshInterval            int           `config:"int;10"`
+	IptablesPostWriteCheckIntervalSecs time.Duration `config:"seconds;1"`
```

Comment:
Will it be clear to the Felix admin/configurer that IptablesPostWriteCheckIntervalSecs can be set to a float, whereas other interval-ish config settings need to be ints?  If not, is there something we can do to make that clearer?